### PR TITLE
successful test of EEPROM reads and writes

### PR DESCRIPTION
**Pull request by g7uhn**
Checked the calculation of VFO EEPROM addresses and offsets carefully.
A minor correction in comments.
Increased the EEPROM write delay a little bit for caution (an extra 5ms never hurt anyone...)
Successful test of getNar(), toggleNar(), toggleBreakIn(), toggleKeyer()

It works! Great stuff Pavel, amazed you've put this together without a radio in front of you!

:-D

### CODE REVIEW

File: lib/ft817/ft817.cpp [hunk 1]
```diff
@@ -505,8 +505,8 @@ bool FT817::writeEEPROM(byte data)
 	getByte();
 
 	// almost all EEPROMs have a write delay, from 1 to 5 msecs
-	// we go here for 5 msec, this must be adjusted in practice
-	delay(5);
+	// we go here for 10 msec, this must be adjusted in practice
+	delay(10);
```
Comment (g7uhn) [inline lib/ft817/ft817.cpp:509]: added a little bit more delay because I'm a cautious fellow...

**Comment by stdevPavelmc** [comment]
Thanks Andy, Let me know when you have the changes ready to merge in this PR.

About making the soft without the radio... I had already done that but with python for chirp software http://chirp.danplanet.com for the BTECH radio family... always with a wonderful help on the other side ;-)

I have the FT-817ND in my wish list from forever... I have reached all FT-817ND owners in Cuba to buy one of them... no luck so far.

**Comment by g7uhn** [comment]
Hi Pavel,

I think that PR is ready to merge into your repo, all the library functions
I use are tested on that.

Today I've been enjoying the tidy code you've created and have been using
it with my ft817_buddy sketch... My ft817_buddy repo has now been rebuilt
with the code organised by PlatformIO.  It's all so much cleaner and easier
to work with now - THANK YOU!!

As I'm working on it now, I've placed a copy of your ft817 library within
my repo (https://github.com/g7uhn/ft817_buddy/tree/development/Arduino/lib)
and I'm now working on adding more functions to the ft817_buddy sketch as
requested by G0POT.  The first addition is a function in the ft817 library
to toggle the RFGain/SQL setting, toggleRfSql().  I've added in the code
for the new function in my local version of your library but I'm wondering
what the best way to link it back to yours is so my additions can flow up
to you.

What's the best way to link the version in my GitHub repo back to yours
while still keeping it within my repo structure?  And then can I create PRs
from my library to yours when I've got a batch of new functions for merging?

I had no idea it's so tough to get electronics into Cuba!  :-O

Once again, thanks for the boost you've given this project!

Cheers,

Andy

**Comment by stdevPavelmc** [comment]
Ok, I will merge the changes ASAP...

Glad to help you and about the library linking, for now we must use it in this way, after the merge I will modify it and make a PR against Arduino official libs to get it included into contributed libs.

Once it is added you will experiment the power of PlatformIO, just declaring the lib name in the platformio.ini file in your working dir it will search/fetch/install it for you.

So contributions will be towards the lib and you will just do in your PlatformIO env `pio update` to get it updated. That's is.

Checking the changes now to merge it.

**Comment by stdevPavelmc** [comment]
>  I had no idea it's so tough to get electronics into Cuba! :-O

This is complicated by not impossible:

- Postal parcel shipping is only good for kits and spare parts, no complete radios allowed.
- Any person (tourist) entering the country can carry a hamradio gear if it's endorsed to a licensed ham here and left in deposit in customs at the airport, no extra charge is made to the carrier (this must be coordinated with the ham inside the country to ease the paperwork, the import charge is assumed by the endorsed hamradio operator)

Any help is appreciated, and thanks beforehand.

**Pull request merged**